DNN Platform/Modules/BulkInstall/BulkInstall.Web/src/components/tabs/bulk-install-install/bulk-install-install.tsx

```diff
@@ -0,0 +1,32 @@
+import { Component, Host, h } from '@stencil/core';
+
+@Component({
+  tag: 'bulk-install-install',
+  styleUrl: 'bulk-install-install.scss',
+  shadow: true,
+})
+export class BulkInstallInstall {
+
+  render() {
+    return (
+      <Host>
+        <div class="row">
+          <div class="col">
+            <div class="panel">
+              <div class="panel-heading">
+                <h3 class="panel-title">Upload Install Package(s)</h3>
+              </div>
+              <div class="panel-body">
+                <dnn-dropzone allowed-extensions="zip"></dnn-dropzone>
```

Comment:
Here we probably want to feed localization values for the inner controls of that component.

Reply:
Yes, I thought you were handling localization.  I need to do it, then I would prefer merging this PR so I can update my environment for all the backend changes you have made and get localization in a working state.